### PR TITLE
Enable drawing donut charts using filled shapes

### PR DESCRIPTION
**Pull request by skovalyov**
* Add donutSolid option support.
* Add solid donut charts examples.

### CODE REVIEW

**Comment by skovalyov** [comment]
By this pull request I would like to revitalize #330, because there was no activity recently. I used the input from the original pull request, implemented maintainer comments and made some changes taking into account the latest code.

File: src/scripts/charts/pie.js [hunk 1]
```diff
@@ -178,21 +183,40 @@
...
-      var path = new Chartist.Svg.Path(!options.donut)
+      var path = new Chartist.Svg.Path(!options.donut || options.donutSolid)
         .move(end.x, end.y)
         .arc(radius, radius, 0, endAngle - startAngle > 180, 0, start.x, start.y);
 
       // If regular pie chart (no donut) we add a line to the center of the circle for completing the pie
       if(!options.donut) {
         path.line(center.x, center.y);
+      } else {
+        if (options.donutSolid) {
```
Comment (gionkunz) [inline src/scripts/charts/pie.js]: Can you make an `else if` from this?

**Review by gionkunz — COMMENTED** [review]
Thanks for the PR! At some point we should also support percentage by using `stripUnit` and calculate relative donut widths. But for the moment this is great! Thanks!

File: src/scripts/charts/pie.js (hunk 1 above)
Comment (skovalyov) [reply src/scripts/charts/pie.js]: Done.

**Comment by gionkunz** [comment]
Awesome, thanks!

**Pull request merged**